FWCore/Integration/plugins/PostInsertProducer.cc

```diff
@@ -0,0 +1,83 @@
+#include "DataFormats/TestObjects/interface/ThingWithDoNotSort.h"
+#include "DataFormats/TestObjects/interface/ThingWithPostInsert.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/global/EDProducer.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/Utilities/interface/EDPutToken.h"
+
+namespace edmtest {
+
+  class PostInsertProducer : public edm::global::EDProducer<> {
```

Comment:
Just to make sure, is this module not intended to be used outside of `FWCore/Integration`?

Reply:
No, I'm only using it in `FWCore/Integration` to test the `post_insert()` functionality.